config/locales/en.txt

```diff
@@ -1477,9 +1477,32 @@
   name_map_no_maps: No observations of [name] have mappable location information.
 
   # observer/news
-  news_title: "Latest Release: August 3, 2015"
-  news_header: This page provides notes on the latest release of the [:app_title] website.
-  news_content: "December 13, 2015. <u>Rails 4.2.5</u>\n\nAt long last the Mushroom Observer website is up to date with the latest stable version of Ruby on Rails (4.2.5). This has not been true since at least March of 2009.  The upgrade process started last August when we were still using the 2.1.1 version of Ruby on Rails.\n\nThis has been a huge team effort by all the volunteer developers on the project. Thanks to every one of you for helping to make this happen.\n\nFor those who are interested in the technical low-down of this specific release, we got to 4.1 last month and the biggest change between 4.1 and 4.2 was a new HTML parsing library that required many minor code changes throughout out code base. There have also been some changes to how email is handled and a variety of other small changes to the code base.  You should see no new changes in the site due to this latest upgrade, but of course <a href=\"/observer/ask_webmaster_question\">send us a note</a> if you see anything wrong."
+  news_title: "Latest Release: August 16, 2015"
```

Comment:
I think you intend this to be 2016.  I tried doing a quick edit of it, but it wanted me to fork your repo.  Hopefully this is an easy fix you can get it quick.